frontend/containers/AuctionsContainer.vue

```diff
@@ -164,6 +167,11 @@ export default Vue.extend({
             this.$store.dispatch('wallet/setup');
             this.$store.dispatch('authorizations/setup');
         },
+        toggleAutoRouterLoad(id: string): void {
+            const previousValue = this.$store.getters['auctions/getAuctionAutoRouterStates'][id] || false;
```

Comment:
Nit: move `|| false` to the getter

Reply:
getter gets the whole dict out. but i guess we can just remove `false` here since the fallback would happen if the value is falsy anyway 🤷🏻 .

Or should i refactor the getter to return the specific value?

Reply:
Technically yes, you can make functional getter. But it's alright, it's just a nit